### PR TITLE
Pp/#979 evaluate a converion from rst to markdown myst

### PR DESCRIPTION
**Pull request by pierrepetersmeier**
!Test

### CODE REVIEW

**Comment by pierrepetersmeier** [comment]
!test

**Comment by pierrepetersmeier** [comment]
!test

File: docs/readthedocs/models/input/participant/bm.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+| Attribute           | Unit    | Remarks                                                   |
+| ------------------- | ------- | --------------------------------------------------------- |
+| uuid                | --      |                                                           |
+| id                  | --      | Human readable identifier                                 |
+| capex               | €       | Capital expenditure to purchase one entity of this type   |
+| opex                | € / MWh | Operational expenditure to operate one entity ofthis type |
+| activePowerGradient | % / h   | Maximum permissible rate of change of power               |
+| sRated              | kVA     | Rated apparent power                                      |
+| cosphiRated         | --      | Rated power factor                                        |
+| etaConv             | %       | Efficiency of the assets inverter                         |
```
Comment (sebastian-peter) [inline docs/readthedocs/models/input/participant/bm.md:20]: There's some grid tables left, like this one.  Is it possible to convert these as well?

**Review by pierrepetersmeier — COMMENTED** [review]
!test

File: docs/readthedocs/models/input/participant/bm.md (hunk 1 above)
Comment (pierrepetersmeier) [reply docs/readthedocs/models/input/participant/bm.md]: Everything should be corrected now

Comment (pierrepetersmeier) [reply docs/readthedocs/models/input/participant/bm.md]: But not. The Links in the new listtables don't work.

**Comment by pierrepetersmeier** [comment]
tables fixed
figures fixed
links in the tables fixed

**Review by pierrepetersmeier — COMMENTED** [review]
tables fixed

File: docs/readthedocs/models/input/grid/gridcontainer.md [hunk 2]
```diff
@@ -37,17 +37,49 @@ A synoptic overview of both classes' attributes is given here:
...
-| predominantVoltageLevel | --   | main voltage level - only `SubGridContainer`      |
-| subnet                  | --   | sub grid number - only `SubGridContainer`         |
+```{eval-rst}
+.. list-table::
+   :widths: 33 33 33
+   :header-rows: 0
 
 
+   * - Attribute
+     - Un it
```
Comment (staudtMarius) [inline docs/readthedocs/models/input/grid/gridcontainer.md:47]: This should be fixed.

File: docs/readthedocs/models/input/grid/line.md [hunk 3]
```diff
@@ -6,38 +6,114 @@ Representation of an AC line.
...
-| b         | µS / km | Phase-to-ground susceptance per length      |
-| iMax      | A       | Maximum permissible current                 |
-| vRated    | kV      | Rated voltage                               |
+```{eval-rst}
+.. list-table::
+   :widths: 33 33 33
+   :header-rows: 0
 
+   * - Attribute
+     - U n i t
```
Comment (staudtMarius) [inline docs/readthedocs/models/input/grid/line.md:15]: Why does this contain empty spaces?

File: docs/readthedocs/models/input/grid/line.md [hunk 4]
```diff
@@ -6,38 +6,114 @@ Representation of an AC line.
...
-| geoPosition       | --   | Line string of geographical locations describing the <br> position of the line                                                                                                         |
-| olmCharacteristic | --   | Characteristic of possible overhead line monitoring <br> Can be given in the form of `olm:{<List of Pairs>}`. <br> The pairs are wind velocity in x and permissible <br> loading in y. |
+```{eval-rst}
+.. list-table::
+   :widths: 33 33 33
+   :header-rows: 0
+
+
+   * - Attribute
+     - U n i t
```
Comment (staudtMarius) [inline docs/readthedocs/models/input/grid/line.md:64]: Same as before.

File: docs/readthedocs/models/input/grid/line.md [hunk 5]
```diff
@@ -6,38 +6,114 @@ Representation of an AC line.
...
+
+   * - nodeB
+     - –
+     -
+
+   * - parallelDevices
+     - –
+     - overall amount of parallel lines to automatically construct
+       (e.g. parallelDevices = 2 will build a total of two lines
+       using the specified parameters)
```
Comment (staudtMarius) [inline docs/readthedocs/models/input/grid/line.md:95]: This text creates a large table width. Can this be modified to reduce the table width?

File: docs/readthedocs/models/input/grid/node.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+     - Timely restriction of operation
+
+   * - vTarget
+     - p.u.
+     - Target voltage magnitude to be used by voltage regulation entities
+
+   * - slack
+     - --
+     - | Boolean indicator, if this nodes serves as a slack node in power
+       | flow calculation
```
Comment (staudtMarius) [inline docs/readthedocs/models/input/grid/node.md:42]: This is rendered as smaller text. Can you fix this?

File: docs/readthedocs/models/input/grid/linegraphic.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+     - Remarks
+
+   * - uuid
+     - --
+     -
+
+   * - graphicLayer
+     - --
+     - | Human readable identifier of the graphic layer to draw
+       | this element on
```
Comment (staudtMarius) [inline docs/readthedocs/models/input/grid/linegraphic.md:26]: Same as in ``nodegraphic.md``.

File: docs/readthedocs/models/input/grid/transformer2w.md [hunk 8]
```diff
@@ -10,41 +10,129 @@ It is assumed, that node A is the node with higher voltage.
...
-| tapNeutr  |      | Neutral tap position                                    |
-| tapMin    |      | Minimum tap position                                    |
-| tapMax    |      | Maximum tap position                                    |
+```{eval-rst}
+.. list-table::
+   :widths: 33 33 33
+   :header-rows: 0
+
+   * - Attribute
+     - Un it
```
Comment (staudtMarius) [inline docs/readthedocs/models/input/grid/transformer2w.md:19]: Same as before.

File: docs/readthedocs/models/input/grid/transformer2w.md [hunk 9]
```diff
@@ -10,41 +10,129 @@ It is assumed, that node A is the node with higher voltage.
...
+   * - gM
+     - nS
+     - No load conductance
+
+   * - bM
+     - nS
+     - No load susceptance
+
+   * - sRated
+     - k VA
```
Comment (staudtMarius) [inline docs/readthedocs/models/input/grid/transformer2w.md:43]: Why is here an empty space?

File: docs/readthedocs/models/input/grid/transformer2w.md [hunk 10]
```diff
@@ -10,41 +10,129 @@ It is assumed, that node A is the node with higher voltage.
...
+   * - tapMin
+     -
+     - Minimum tap position
+
+   * - tapMax
+     -
+     - Maximum tap position
+```
 
 A list with some standard transformer types can be found here: `Standard Two Winding Transformer Types`_
```
Comment (staudtMarius) [inline docs/readthedocs/models/input/grid/transformer2w.md:80]: Here should be a link. See: ``line.md``
There are also two additional broken links below.

File: docs/readthedocs/models/input/grid/transformer2w.md [hunk 11]
```diff
@@ -10,41 +10,129 @@ It is assumed, that node A is the node with higher voltage.
...
-| type            | --   |                                                                                                                                                                               |
-| tapPos          | --   | Current position of the tap changer                                                                                                                                           |
-| autoTap         | --   | true, if there is a tap regulation apparent and active                                                                                                                        |
+```{eval-rst}
+.. list-table::
+   :widths: 33 33 33
+   :header-rows: 0
+
+   * - Attribute
+     - U n i t
```
Comment (staudtMarius) [inline docs/readthedocs/models/input/grid/transformer2w.md:91]: Same as before.

File: docs/readthedocs/models/input/grid/transformer2w.md [hunk 12]
```diff
@@ -10,41 +10,129 @@ It is assumed, that node A is the node with higher voltage.
...
 
+   * - nodeB
+     - –
+     - Lower voltage node
+
+   * - parallelDevices
+     - –
+     - overall amount of parallel transformers to automatically
+         construct (e.g. parallelDevices = 2 will build a total of
+         two transformers using the specified parameters)
```
Comment (staudtMarius) [inline docs/readthedocs/models/input/grid/transformer2w.md:122]: This is not rendered correctly in the docs.

File: docs/readthedocs/models/input/grid/transformer3w.md [hunk 13]
```diff
@@ -0,0 +1,186 @@
...
+
+   * - nodeC
+     - --
+     - Lowest voltage node
+
+   * - parallelDevices
+     - --
+     - | overall amount of parallel transformers to automatically
+       | construct (e.g. parallelDevices = 2 will build a
+       | total of two transformers using the specified parameters)
```
Comment (staudtMarius) [inline docs/readthedocs/models/input/grid/transformer3w.md:165]: This is smaller than the other text.

File: docs/readthedocs/models/input/participant/bm.md [hunk 14]
```diff
@@ -0,0 +1,115 @@
...
+     - Human readable identifier
+
+   * - capex
+     - €
+     - Capital expenditure to purchase one entity of this type
+
+   * - opex
+     - € / MWh
+     - | Operational expenditure to operate one entity of
+       | this type
```
Comment (staudtMarius) [inline docs/readthedocs/models/input/participant/bm.md:36]: Same as before.

File: docs/readthedocs/models/input/participant/bm.md [hunk 15]
```diff
@@ -0,0 +1,115 @@
...
+   * - marketReaction
+     - --
+     - | Whether to adapt output based on (volatile)
+       | market price or not
+
+   * - costControlled
+     - --
+     - | Whether to adapt output based on the difference
+       | between production costs and fixed feed in tariff or not
+
```
Comment (staudtMarius) [inline docs/readthedocs/models/input/participant/bm.md:105]: Same as before.

File: docs/readthedocs/models/input/participant/chp.md [hunk 16]
```diff
@@ -0,0 +1,122 @@
...
+     - Human readable identifier
+
+   * - capex
+     - €
+     - Capital expenditure to purchase one entity of this type
+
+   * - opex
+     - € / MWh
+     - | Operational expenditure to operate one entity of
+       | this type
```
Comment (staudtMarius) [inline docs/readthedocs/models/input/participant/chp.md:36]: Same as before.

File: docs/readthedocs/models/input/participant/ev.md [hunk 17]
```diff
@@ -0,0 +1,97 @@
...
+     - Human readable identifier
+
+   * - capex
+     - €
+     - Capital expenditure to purchase one entity of this type
+
+   * - opex
+     - € / MWh
+     - | Operational expenditure to operate one entity of
+       | this type
```
Comment (staudtMarius) [inline docs/readthedocs/models/input/participant/ev.md:36]: Same as before.

File: docs/readthedocs/models/input/participant/hp.md [hunk 18]
```diff
@@ -0,0 +1,102 @@
...
+
+   * - capex
+     - €
+     - Capital expenditure to purchase one entity of this type
+
+   * - opex
+     - € / MWh
+     - | Operational expenditure to operate one entity of
+       | this type
+
```
Comment (staudtMarius) [inline docs/readthedocs/models/input/participant/hp.md:37]: Same as before.

File: docs/readthedocs/models/input/participant/load.md [hunk 19]
```diff
@@ -0,0 +1,89 @@
...
+If no matching standard load profile is known, {code}`LoadProfile#NO_LOAD_PROFILE` can be used.
+
+To assist the user in marking the desired load profile, the enum {code}`BdewLoadProfile` provides a collection of
+commonly known German standard electricity load profiles, defined by the bdew (Bundesverband der Energie- und
+Wasserwirtschaft; engl. Federal Association of the Energy and Water Industry). For more details see
+[the corresponding website (German only)](https://www.bdew.de/energie/standardlastprofile-strom/).
+
+Furthermore there are {code}`TemperatureDependantLoadProfiles` which can be used to note usage of load profiles for night heating storages or heat pumps for example.
+The profiles rely on the VDN description for interruptable loads.
+For more details see [here (German only)](https://www.bdew.de/media/documents/LPuVe-Praxisleitfaden.pdf/).
```
Comment (staudtMarius) [inline docs/readthedocs/models/input/participant/load.md:87]: This site does not exist.

File: docs/readthedocs/models/input/participant/pv.md [hunk 20]
```diff
@@ -0,0 +1,84 @@
...
+     - :ref:`Reactive power characteristic<participant-general-q-characteristic>` to follow
+
+   * - albedo
+     - --
+     - `Albedo <https://en.wikipedia.org/wiki/Albedo>`_ of the plant's surrounding
+
+   * - azimuth
+     - °
+     - | Inclination in a compass direction
+       | South = 0°, West = 90°, East = -90°
```
Comment (staudtMarius) [inline docs/readthedocs/models/input/participant/pv.md:48]: Same as before.

File: docs/readthedocs/models/input/participant/pv.md [hunk 21]
```diff
@@ -0,0 +1,84 @@
...
+     - Generator correction factor merging technical influences
+
+   * - kT
+     - --
+     - Temperature correction factor merging thermal influences
+
+   * - marketReaction
+     - --
+     - | Whether to adapt output based on (volatile)
+       | market price or not
```
Comment (staudtMarius) [inline docs/readthedocs/models/input/participant/pv.md:69]: Same as before.

File: docs/readthedocs/models/input/participant/storage.md [hunk 22]
```diff
@@ -0,0 +1,129 @@
...
+     - Human readable identifier
+
+   * - capex
+     - €
+     - Capital expenditure to purchase one entity of this type
+
+   * - opex
+     - € / MWh
+     - | Operational expenditure to operate one entity of
+       | this type
```
Comment (staudtMarius) [inline docs/readthedocs/models/input/participant/storage.md:36]: Same as before.

File: docs/readthedocs/models/input/participant/storage.md [hunk 23]
```diff
@@ -0,0 +1,129 @@
...
+     - Rated apparent power
+
+   * - cosPhiRated
+     - --
+     - Rated power factor
+
+   * - pMax
+     - kW
+     - | Maximum permissible active power
+       | infeed or consumption
```
Comment (staudtMarius) [inline docs/readthedocs/models/input/participant/storage.md:53]: Same as before.

File: docs/readthedocs/models/input/participant/storage.md [hunk 24]
```diff
@@ -0,0 +1,129 @@
...
+     - Maximum permissible rate of change of power
+
+   * - eta
+     - %
+     - Efficiency of the electrical inverter
+
+   * - dod
+     - %
+     - | Maximum permissible depth of discharge. 80 % dod
+       | is equivalent to a state of charge of 20 %.
```
Comment (staudtMarius) [inline docs/readthedocs/models/input/participant/storage.md:66]: Same as before.

File: docs/readthedocs/models/input/participant/storage.md [hunk 25]
```diff
@@ -0,0 +1,129 @@
...
+     - :ref:`Reactive power characteristic<participant-general-q-characteristic>` to follow
+
+   * - type
+     - --
+     -
+
+   * - behaviour
+     - --
+     - | Foreseen operation strategy of the storage.
+       | Eligible input: *"market"*, *"grid"*, *"self"*
```
Comment (staudtMarius) [inline docs/readthedocs/models/input/participant/storage.md:121]: Same as before.

File: docs/readthedocs/models/input/participant/thermalbus.md [hunk 26]
```diff
@@ -0,0 +1,44 @@
+(thermal-bus-model)=
+
+# Thermal Bus
+
+A coupling point to thermal system - equivalent to [electrical node](https://powersystemdatamodel.readthedocs.io/en/latest/models.md/result.md/grid.md/node.md#node).
```
Comment (staudtMarius) [inline docs/readthedocs/models/input/participant/thermalbus.md:5]: This link is broken.

File: docs/readthedocs/models/input/participant/wec.md [hunk 27]
```diff
@@ -0,0 +1,121 @@
...
+     - Human readable identifier
+
+   * - capex
+     - €
+     - Capital expenditure to purchase one entity of this type
+
+   * - opex
+     - € / MWh
+     - | Operational expenditure to operate one entity of
+       | this type
```
Comment (staudtMarius) [inline docs/readthedocs/models/input/participant/wec.md:36]: Same as before.

File: docs/readthedocs/models/input/participant/wec.md [hunk 28]
```diff
@@ -0,0 +1,121 @@
...
+     - :ref:`Reactive power characteristic<participant-general-q-characteristic>` to follow
+
+   * - type
+     - --
+     -
+
+   * - marketReaction
+     - --
+     - | Whether to adapt output based on (volatile)
+       | market price or not
```
Comment (staudtMarius) [inline docs/readthedocs/models/input/participant/wec.md:107]: Same as before.

**Review by staudtMarius — CHANGES_REQUESTED** [review]
Here are some remoarks from my side.

File: docs/readthedocs/models/input/grid/node.md (hunk 6 above)
Comment (sebastian-peter) [reply docs/readthedocs/models/input/grid/node.md:42]: I noticed this before as well when I worked with list tables, don't know if it is easily fixable. Wouldn't spend too much time on this tbh, but you can still try if you like

File: docs/readthedocs/models/input/grid/gridcontainer.md (hunk 2 above)
Comment (pierrepetersmeier) [reply docs/readthedocs/models/input/grid/gridcontainer.md]: I hope I have eliminated all attention errors.

File: docs/readthedocs/models/input/grid/line.md (hunk 5 above)
Comment (pierrepetersmeier) [reply docs/readthedocs/models/input/grid/line.md]: Now we have the smaller text again.

File: docs/readthedocs/models/input/participant/thermalbus.md (hunk 26 above)
Comment (pierrepetersmeier) [reply docs/readthedocs/models/input/participant/thermalbus.md]: Check

File: docs/readthedocs/models/input/grid/transformer2w.md (hunk 12 above)
Comment (pierrepetersmeier) [reply docs/readthedocs/models/input/grid/transformer2w.md]: Check

File: docs/readthedocs/models/input/participant/load.md (hunk 19 above)
Comment (pierrepetersmeier) [reply docs/readthedocs/models/input/participant/load.md]: I found this Link. https://www.bdew.de/media/documents/LPuVe-Praxisleitfaden.pdf

File: docs/readthedocs/models/input/grid/node.md (hunk 6 above)
Comment (pierrepetersmeier) [reply docs/readthedocs/models/input/grid/node.md:42]: I haven't found anything concrete for the solution. It appears consistent, though, as it gets smaller with each line break.

**Review by pierrepetersmeier — COMMENTED** [review]
I hope all errors are corrected now.

File: docs/readthedocs/models/input/grid/node.md (hunk 6 above)
Comment (staudtMarius) [reply docs/readthedocs/models/input/grid/node.md:42]: OK. I think we can keep these part as they are now.

**Review by pierrepetersmeier — COMMENTED** [review]
!test

**Review by staudtMarius — APPROVED** [review]
Looks good to me now.

**Pull request merged**